Source/UnitTests/VideoCommon/VertexLoaderTest.cpp

```diff
@@ -72,14 +72,15 @@ class VertexLoaderTest : public testing::Test
     m_src.Write<T, true>(val);
   }
 
-  void ExpectOut(float val)
+  void ExpectOut(float expected)
   {
     // Read unswapped.
-    Common::IntFloat expected(val), actual(m_dst.Read<float, false>());
-    if (!actual.f || actual.f != actual.f)
-      EXPECT_EQ(expected.i, actual.i);
+    const float actual = m_dst.Read<float, false>();
+
+    if (!actual || actual != actual)
```

Comment:
Unrelated to your PR, but this condition looks a bit... weird.

Reply:
`float != float` -> use the internal bit representation for comparison (in case its a NaN or whatever)